app/services/cart_create_service.rb

```diff
@@ -8,7 +8,7 @@ def initialize(current_user, params_request, params)
   end
 
   def call(current_order)
-    @current_order = current_order
+    @order = current_order
```

Comment:
What about case when current_order is nil?
We discussed that. Please revise that.